ape_avalanche/ecosystem.py

```diff
@@ -9,10 +11,27 @@
 }
 
 
+def _create_network_config(
+    required_confirmations: int = 1, block_time: int = 2, **kwargs
+) -> NetworkConfig:
+    # Helper method to isolate `type: ignore` comments.
```

Comment:
Type ignore may no longer be needed

Reply:
you may have to make this change https://github.com/ApeWorX/ape-optimism/blob/main/pyproject.toml#L6 and similar ones first, and you can run `mypy . --warn-unused-ignores`